composer/loggers/file_logger.py

```diff
@@ -43,6 +43,15 @@ class FileLogger(LoggerCallback):
                 loggers=[logger]
             )
 
+        .. testcleanup::
+
+            import os
+            from composer.utils.run_directory import get_run_directory
+
+            path = os.path.join(get_run_directory(), "log.txt")
+            if os.path.exists(path):
+                os.remove(path)
```

Comment:
we are very unlikely to encounter this, but would prefer a try / except here to avoid the race condition